db/migrate/20181105212016_duplicate_deleted_at_state_changed_at.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class DuplicateDeletedAtStateChangedAt < ActiveRecord::Migration
+  def up
+    begin
+      Account.transaction do
+        Account.scheduled_for_deletion.where.has{ deleted_at != nil}.find_each do |account|
+          account.update({state_changed_at: account.deleted_at})
+        end
+      end
+    rescue
+      raise ActiveRecord::Rollback
```

Comment:
That raise is not populated above so the transaction is rollbacked but the migration is still successfull